codeworld-compiler/src/CodeWorld/Compile/Framework.hs

```diff
@@ -305,32 +311,95 @@ ghcParseCode extraExts src = do
       state = GHC.mkPState dflagsWithPragmas buffer location
   return $ case GHC.unP GHC.parseModule state of
     GHC.POk _ (GHC.L _ mod) -> GHCParsed mod
-    GHC.PFailed _ _ _ -> GHCNoParse
+    GHC.PFailed _ -> GHCNoParse
 
 fakeDynFlags :: GHC.DynFlags
 fakeDynFlags = GHC.defaultDynFlags fakeSettings fakeLlvmConfig
 
+fakePlatformMisc :: GHC.PlatformMisc
+fakePlatformMisc = GHC.PlatformMisc
```

Comment:
I almost cringe to ask it, but can you include field names here so the reader has some hope of finding what they need to change in the future?